src/views/Home.vue

```diff
@@ -1,28 +1,94 @@
 <template>
   <div>
-    <div v-for="restaurant in restaurants" v-bind:key="restaurant.id">
-     <h1>
-       <router-link :to="'/restaurant/'+restaurant.slug">{{restaurant.title}}</router-link>
-      </h1>
+    <div class="banner">
+      <ChipSearchbar @search="search"></ChipSearchbar>
+    </div>
+    <div v-if="results.length > 0 && !isLoading" class="results">
+      <h4 class="text--secondary" v-if="results.length === 1" >1 result was found</h4>
```

Comment:
Why is it necessary to add conditional logic for display of one? If it is because of plural on "result", add the condition on the word itself instead of doing on the entire title.

Reply:
That is how it currently is, this is an older version